dockstore-webservice/src/main/java/io/dockstore/webservice/helpers/SemVer.java

```diff
@@ -0,0 +1,40 @@
+package io.dockstore.webservice.helpers;
+
+/**
+ * This class is used to compare between versions defined using semantic versioning (SemVer)
```

Comment:
We already have a library for this, might as well re-use it. 

[INFO] io.dockstore:dockstore-file-plugin-parent:jar:1.5.0-SNAPSHOT
[INFO] +- ro.fortsoft.pf4j:pf4j:jar:1.1.1:provided
[INFO] |  +- org.slf4j:slf4j-api:jar:1.7.25:provided
[INFO] |  \- com.github.zafarkhaja:java-semver:jar:0.9.0:provided